docs/ares_threadsafety.3

```diff
@@ -0,0 +1,37 @@
+.\"
+.\" SPDX-License-Identifier: MIT
+.\"
+.TH ARES_REINIT 3 "26 November 2023"
+.SH NAME
+ares_threadsafety \- Query if c-ares was built with thread-safety
+.SH SYNOPSIS
+.nf
+#include <ares.h>
+
+ares_bool_t ares_threadsafety(void);
+.fi
+.SH DESCRIPTION
+The \fBares_threadsafety(3)\fP function returns if the library was built with
+thread safety enabled or not.
+
```

Comment:
Maybe this description could also clarify somewhat what it means and what you can do with the library when it is thread-safe vs not thread-safe?